release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -793,16 +844,43 @@ This dashboard provides API server metrics, such as:
 
 // TODO: Link to troubleshooting using this dashboard section once it is available
 
-[id="ocp-4-7-monitoring-etcd-alerts"]
-==== New etcd alerts
+[id="ocp-4-7-monitoring-namespace-by-pod-and-pod-total-Grafana-dashboards"]
+==== Enable Namespace (Pods) and Pod Kubernetes networking dashboards in Grafana
+
+The *Namespace (Pods)* and *Pod* Kubernetes networking dashboards are now enabled in Grafana. You can access the *Namespace (Pods)* and *Pod* dashboards from the web console by navigating to *Monitoring* -> *Dashboards* -> *Grafana UI*.
```

Comment:
Is this available in the **Administrator** and **Developer** perspectives? 

If it is only available in the **Administrator** perspective, then you could end the sentence with "...in the **Administrator** perspective."

If this is available in both, you could add details about how to access the Grafana UI in each respective perspective, maybe as bullet points.

Reply:
Admin perspective only.